debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorTask.java

```diff
@@ -163,6 +163,14 @@ public synchronized void start(Map<String, String> props) {
                     readers.add(binlogReader);
                 }
             } else {
+                if (!source.hasFilterInfo()) {
```

Comment:
Wouldn't we have to set it to the current filter info in any case? It seems not used yet at this point, so it probably doesn't make a difference, so just in case.